src/com/jalasoft/search_files/search/asset.py

```diff
@@ -0,0 +1,40 @@
+#import File
+import os
+
+class Asset (object):
+
+    def __init__(self, path, is_file):
+        self._path = path
+        self._is_file = is_file
+        self._size = os.path.getsize(path)
+
+    def get_path(self):
+        return self._path
+
+    def get_size(self):
+        return self._size
+
+    def get_is_file(self):
+        return self._is_file
+
+class File(Asset):
```

Comment:
Please add a new line before of the Class definition line, it should contains 2 lines before of it